src/functionalTest/java/uk/gov/hmcts/reform/bulkscanprocessor/controllers/ProcessedEnvelopeMessageHandlingTest.java

```diff
@@ -67,6 +68,11 @@ public void should_complete_envelope_referenced_by_queue_message() throws Except
         EnvelopeResponse updatedEnvelope = getEnvelope(zipFilename);
         assertThat(updatedEnvelope.getScannableItems()).hasSize(2);
         assertThat(updatedEnvelope.getScannableItems()).allMatch(item -> item.ocrData == null);
+
+        await("file should be deleted")
+            .atMost(scanDelay + 40_000, TimeUnit.MILLISECONDS)
```

Comment:
`scanDelay` doesn't apply here - it's the delay between two consecutive runs of blob processing job.

Worth, by analogy to how it's done elsewhere, introducing a constant representing maximum allowed time for completed blob deletion (probably even much shorter, as deletion job runs every 10 seconds).

Reply:
Also, just before `UUID envelopeId = getEnvelope(zipFilename).getId();` we need to make an assertion checking that file exists.

Reply:
Having had more thoughts, we can't test the new deletion process here, as this test will only pass because of the old process deleting the file.

This assertion then duplicates the test which has been deleted. I think we shouldn't make changes to these existing tests (those verifying that file has been deleted) at this stage and do it only when deletion bit has been removed from the main job. As long as the main job is responsible for deletion, that should be covered by tests.